projects/cdk/classes/control.ts

```diff
@@ -71,7 +72,9 @@ export abstract class TuiControl<T> implements ControlValueAccessor {
                 map(() => this.control.control),
                 filter(Boolean),
                 distinctUntilChanged(),
-                switchMap((c) => merge(c.valueChanges, c.statusChanges)),
+                switchMap((c) =>
```

Comment:
`as any` so sad (( why not `as unknown as what-do-you-need`

Reply:
Because it makes no difference and is just longer